src/polar/Pie.tsx

```diff
@@ -391,117 +400,107 @@ const renderLabelItem = (option: PieLabel, props: any, value: any) => {
   );
 };
 
-export class Pie extends PureComponent<InternalProps, State> {
-  pieRef: SVGGElement = null;
-
-  sectorRefs: SVGGElement[] = [];
-
-  static displayName = 'Pie';
-
-  static defaultProps = {
-    stroke: '#fff',
-    fill: '#808080',
-    legendType: 'rect',
-    cx: '50%',
-    cy: '50%',
-    startAngle: 0,
-    endAngle: 360,
-    innerRadius: 0,
-    outerRadius: '80%',
-    paddingAngle: 0,
-    labelLine: true,
-    hide: false,
-    minAngle: 0,
-    isAnimationActive: !Global.isSsr,
-    animationBegin: 400,
-    animationDuration: 1500,
-    animationEasing: 'ease',
-    nameKey: 'name',
-    blendStroke: false,
-    rootTabIndex: 0,
+export function computePieSectors({
+  pieSettings,
+  displayedData,
+  cells,
+  offset,
+}: {
+  displayedData: any[];
```

Comment:
Should this be `RealPieData[]`? Since it's expecting the result of `getRealPieData`

Reply:
`RealPieData` is `any[]` so technically yes 😅